frontend/src/pages/Rubrics/RubricEditor/index.tsx

```diff
@@ -29,6 +31,7 @@ const RubricEditor: React.FC = () => {
     const navigate = useNavigate();
 
     const { selectedClassroom } = useContext(SelectedClassroomContext)
+    useClassroomUser(selectedClassroom?.id, ClassroomRole.PROFESSOR, "/access-denied");
```

Comment:
Is there a way to do this within the `contexts/selectedClassroom.tsx` file rather than repeating it everywhere? There may not be but could be cleaner.

Reply:
The reason I have this repeated is b/c on some pages, we don't want to go to access denied (for example, the landing page). I think it makes sense to keep it in useClassroomUser b/c that is the hook we use to get the classroom role of the current user. selectedClassroomContext doesn't worry about the role of the user currently